assets/src/parse_bundle_plugin.js

```diff
@@ -44,13 +44,8 @@ var parseBundlePlugin = function(data, base_dir) {
             // instantiated as a global variable. Only currently used by the Kolibri core app.
             external = data.name;
             // change the periods of the Python module path name to underscores, so that it is a valid JS variable name.
-            library = data.core ? 'Kolibri' : data.name.replace(/\./g, "_");
+            library = data.core ? 'kolibriGlobal' : data.name.replace(/\./g, "_");
```

Comment:
What is the motivation for this name change? It seems a little redundant to call it `Global` given that it is already set on the `global`/`window` object?